packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -90,6 +90,10 @@ class OverlayEntry {
     _overlay._didChangeEntryOpacity();
   }
 
+  void makeOpaqueHack() {
```

Comment:
I can't just call the opaque setter as it triggers side effects that crash given the place I am naively forcing the opacity of the layer. I imagine the ideal fix would either be to start the appropriate OverlayEntry as opaque to begin with, tweak this method to not crash in this case, or to simulate exactly the same code path that is followed at the end of  transitioning between routes when initializing routes.